app/controllers/chat_channels_controller.rb

```diff
@@ -262,6 +262,31 @@ def change_status
     render json: success_json
   end
 
+  def delete
```

Comment:
```suggestion
  def destroy
```

We mainly use `destroy` in our codebase. Perhaps worth standardising here?

Reply:
Ah yeah my bad, will change